### ISSUE TITLE
Creating the dev section theme

### ISSUE THREAD

**Issue by MatthewsSam** [konstin/stk-wiki-theme#18]
It's a minor thing but when we did the concepts for the wiki, general public article had the normal header while documentations and more in depth article where with a violet header.

I will do tomorrow the header so we can include it. Not sure how to differentiate general article versus documentation.

![peeyz4d](https://cloud.githubusercontent.com/assets/6406045/9565077/a422832a-4eba-11e5-8691-b719c246453a.jpg)

**Comment by MTres19** [comment]
I think this could be accomplished with magic words. The real question: is it worth it?

**Comment by konstin** [comment]
As it's pretty easy do to (at leaast for my part), I'ld say yes.

@samuncle: Is the dev section ready?

**Comment by MTres19** [comment]
Actually it could simply be done with a template including CSS to override
the header background image using <pre>!important</pre>.

**Comment by konstin** [comment]
It would require some awful hacks to do Page-name dependent css.

**Comment by MTres19** [comment]
Exactly. So one would just transclude the template into any page that
should use the dev theme.

**Comment by konstin** [comment]
I'ld prefer a clean and maintainable solution inside the skin written on a few lines of simple PHP. Doing this in a template violates every principle of good application design.

**Comment by MTres19** [comment]
Sure, whatever you want.
